studio/src/components/QDataArea/QDataArea.jsx

```diff
@@ -0,0 +1,409 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable react/prop-types */
+/* eslint-disable no-param-reassign */
+import 'regenerator-runtime/runtime'; // Polyfill for using async/await
```

Comment:
Would be better to bring this polyfill in on a higher level - `studio/src/index.js` so that async/await can be used throughout the whole project for consistency.

Reply:
You can use at any level, just import it where you want to use it.